archivessnake/create_daos.py

```diff
@@ -56,6 +58,26 @@ def main(resource, series, dip_file):
             print(e)
 
 
+def get_candidate_dips(am_client, project_name, folder_end):
+    '''Gets all DIPs from the Archivematica Storage Service and returns DIPs that match specific criteria.
+
+    Args:
+        am_client (am_client.AMClient.am_client): instantiation of am_client
+        project_name (str): collection or other name expected to be part of SIP name
+        folder_end (int): assumes "Folder" is in DIP name, folder # to stop at
```

Comment:
I would suggest documenting this assumption in a more general docstring, and also adding that the assumption is that the string `Folder` is followed by an integer only (i.e. based on the code below something like `Folder12a` would not work).

Reply:
@bonniegee did you see this comment?

Reply:
Looking at this again, I'm realizing the change I made is to add `(int)`, in which case it must be a number.

Reply:
OK got it, that makes sense!